src/services/configService.js

```diff
@@ -69,4 +63,22 @@ export class ConfigService {
     return HttpClient.put(`${ENDPOINT}solution-settings/logo`, logo, options)
       .map(prepareLogoResponse);
   }
+
+  /* get solution settings */
+  static getSolutionSettings() {
+    return HttpClient.get(`${ENDPOINT}solution-settings/theme`)
```

Comment:
Theme?

Reply:
no

Reply:
I thought you were asking if it isn't "Theme" in camel case. The URL has a weird name in API side. But this is the correct one.